lib/gui/pages/finish/styles/_finish.scss

```diff
@@ -26,7 +26,9 @@
 }
 
 .page-finish .button-primary {
-  min-width: $btn-min-width;
```

Comment:
We shouldn't just stop of using this variable. We either update it, so all buttons have the same min width, or adjust the banner's buttons to match the application.